src/test/kotlin/net/maizegenetics/phgv2/utils/CheckSum.kt

```diff
@@ -0,0 +1,40 @@
+package net.maizegenetics.phgv2.utils
+
+import java.io.FileInputStream
+import java.io.InputStream
+import java.security.MessageDigest
+
+/**
+ * Allows user to specify the protocol, e.g. MD5, SHA-1, SHA-256
+ *
+ * @param filename filename to checksum
+ * @param protocol protocol
+ *
+ * @return check sum
+ */
+fun getChecksum(filename: String, protocol: String = "MD5"): String {
+    try {
+
+        val inputStream: InputStream = FileInputStream(filename)
+        val digester = MessageDigest.getInstance(protocol)
+        val buffer = ByteArray(8192)
+        var numOfBytesRead: Int
+        while (inputStream.read(buffer).also { numOfBytesRead = it } > 0) {
+            digester.update(buffer, 0, numOfBytesRead)
+        }
+        val hashValue = digester.digest()
+        return convertBytesToHex(hashValue)
+    } catch (ex: Exception) {
+        println(ex.message)
+        throw ex
+    }
+
+}
```

Comment:
Terry - this method already lives in VariantLoadingUtils.kt:getChecksumForString().  We probably shouldn't have it in 2 places.  If we keep it here, then it should be removed from VariantLoadingUtils.kt (I don't care which version is kept, but we should consistently use just 1)

Reply:
oh, I see.  Yes, we probably need both and it would be good to have them in the same file.